src/risknet/main.py

```diff
@@ -66,10 +66,13 @@ def run_job(args: Namespace, job_args: Dict[str, str]) -> None:
 
 
 if __name__ == "__main__":
+    #Currently only the following modules can be run from main : label_prep.py,pipeline.py
+    #add an execute function to a module to make it runnable via main entrypoint. 
     args = get_args()
-
+    print(args)
     job_args, env = get_job_args(args)
-
+    
+    print('job_args',job_args)
```

Comment:
can we make this a logger.info statement instead of print

Reply:
Note that the reason for logs is that they don't just get printed to the console, they can get timestamped and written to a .log file so they're persistent. Definitely useful for keeping track of the model lifecycle.